### PR TITLE
feature- loading indicators

### PR DESCRIPTION
**Pull request by sjacobs1**
##Description
added loading indicators

##GIF
ezgif com-optimize-5

##Checklist
- [x]  The code follows the project's coding standards.
- [ ]  100% Unit test coverage for Tested View Models
- [x]  All warnings Resolved.

### CODE REVIEW

File: MovieAppPetProject/MovieAppPetProject/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -91,6 +91,7 @@
       "size" : "83.5x83.5"
     },
     {
+      "filename" : "Screenshot 2024-05-30 at 11.52.45 AM.jpg",
```
Comment (kieran-woodrow) [inline MovieAppPetProject/MovieAppPetProject/Assets.xcassets/AppIcon.appiconset/Contents.json]: please name this asset better

File: MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift [hunk 2]
```diff
@@ -67,11 +66,24 @@ class NewHomeViewController: UIViewController, UITableViewDelegate, UITableViewD
             movieDetailsVC.setMovieID(movieID: movieId)
         }
     }
+
+    private func setupActivityIndicator() {
+        activityIndicator.hidesWhenStopped = true
+    }
+
+    private func loadMovies() {
+        activityIndicator.startAnimating()
```
Comment (kieran-woodrow) [inline MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift]: do this in a delgate

**Review by kieran-woodrow — CHANGES_REQUESTED** [review]

**Review by kieran-woodrow — DISMISSED** [review]

File: MovieAppPetProject/MovieAppPetProject/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 1 above)
Comment (sjacobs1) [reply MovieAppPetProject/MovieAppPetProject/Assets.xcassets/AppIcon.appiconset/Contents.json]: will do

File: MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift (hunk 2 above)
Comment (sjacobs1) [reply MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift]: moved to delegate

**Review by kieran-woodrow — APPROVED** [review]

**Review by MarcusNkadimeng — APPROVED** [review]

**Pull request merged**